SoftLayer/CLI/block/replication/disaster_recovery_failover.py

```diff
@@ -0,0 +1,32 @@
+"""Failover an inaccessible file volume to its available replicant volume."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer.CLI import exceptions
+
+
+@click.command()
+@click.argument('volume-id')
+@click.option('--replicant-id', help="ID of the replicant volume")
+@environment.pass_env
+def cli(env, volume_id, replicant_id):
+    """Failover an inaccessible file volume to its available replicant volume."""
```

Comment:
Might want to mention the dangers of running this command here. And maybe mention to use `replica-failover` for most cases?